_lessons/2022-04-07-anticosti-historical-data-tutorial/anticosti-historical-data-tutorial.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Anticosti Historical Data Tutorial"
+title: "After the storm: Impacts of a hemlock looper outbreak on tree species in Anticosti Island, 1973"
```

Comment:
Maybe the title could be shorter and the new title you added would go in the "description" filed... What do you think? I really liked it, by the way!

Reply:
@graciellehigino - I think for the actual tutorial, it would be best to keep the longer descriptive title, but for the webpage's entry I think the short tutorial sounds great! What do you think @quantitative-ecologist ?

Reply:
I don't really mind. It's nice to have a long title to convey the story, but I think that for the tutorial in general, the short title would be good for the website as you suggested @vmglynn.